media/doc/Romanian translation notes.txt

```diff
@@ -48,9 +48,7 @@ La acestea se adaugă și câteva norme (deduse) din „greșeli frecvente” ra
   - În cazul unor subiecte unice (dintr-un context local sau global), este folosită forma articulată. Exemple: „Calculatorul meu” (numai unul, conținut concret), „Sistemul de operare” (unic în cadrul său), „Documentele mele” (conținut concret) vs. „Locații în rețea” (nedefinit), „Linie de comandă” (cu multiple posibile instanțe distincte/independente), etc.
   20. În relația calculator - utilizator, la raportarea de informații, sunt folosite verbe la diateza pasivă (nu reflexivă). Astfel, pentru „Receiving data from %s” vor fi greșite atât „Primesc date de la %s” cât și „Se primesc date de la %s”, deoarece calculatorul nu este un actor ci un instrument, precum și, cu rare excepții, ca spre exemplu în cazul erorilor care pot fi considerate a avea capacitatea de a surveni în execuția unui proces, datele sau alte obiecte nu au nici o capacitate de acțiune, nici măcar de una reflexivă. Corect rămâne „Sunt primite/recepționate date de la %s”.
 
-[1] http://i18n.ro/Ghidul_traducătorului_de_software
-[2] https://msdn.microsoft.com/en-us/library/ms971323.aspx#atg_keyboardshortcuts_selecting_access_keys
-[3] http://i18n.ro/Greșeli_frecvente
-[4] http://groups.google.com/group/diacritice
```

Comment:
Fortunately for the `ii8n.ro` links there is Wayback Machine with cached copies of saved website pages such as this one from 2017 -> https://web.archive.org/web/20170307094948/http://i18n.ro/Ghidul_traduc%C4%83torului_de_software

I would argue that you remove the original links as you do now but instead replace them with those from cached entries from Wayback Machine.

Reply:
I replaced the links with the cached ones and I added them as reference numbers.

Reply:
> Fortunately for the `ii8n.ro` links there is Wayback Machine with cached copies of saved website pages such as this one from 2017 -> https://web.archive.org/web/20170307094948/http://i18n.ro/Ghidul_traduc%C4%83torului_de_software
> 
> I would argue that you remove the original links as you do now but instead replace them with those from cached entries from Wayback Machine.

That particular resource can also be found here: [https://diacritice.sourceforge.io/ghid.html](https://diacritice.sourceforge.io/ghid.html)
...but I also prefer Wayback Machine, for it is more reliable, although a bit slower.

Reply:
I can add both sources if you want.

Reply:
> I can add both sources if you want.

I don't, really. ☺